etc/config/csharp.defaults.properties

```diff
@@ -0,0 +1,17 @@
+compilers=csharp
+supportsBinary=false
+needsMulti=false
+compilerType=csharp
+
+compiler.csharp.exe=/opt/compiler-explorer/dotnet-6.0/.dotnet/dotnet
```

Comment:
As per https://github.com/compiler-explorer/misc-builder/commit/cb00784922c3083c54237d448391570b15ddc597, should I change this path from `dotnet-6.0` to `dotnet-v6.0.0`?

Reply:
most likely